build_tools/kokoro/gcp_ubuntu/cmake/baremetal/riscv32/test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Test the cross-compiled RISCV targets using Kokoro.
+
+set -e
+set -x
+
+# Print the UTC time when set -x is on
+export PS4='[$(date -u "+%T %Z")] '
+
+# Docker image has the QEMU installed at /usr/src/qemu-riscv.
```

Comment:
Should we make this an env variable? That would make it possible to run this script not within our specific docker images, which is useful (that's why we have CMAKE_BIN as an env variable, for instance)

Reply:
I will create a docker cleanup PR with both env variable updates.